src/state/slices/portfolioSlice/selectors.ts

```diff
@@ -972,13 +978,52 @@ export const selectValidatorIds = createDeepEqualOutputSelector(
   },
 )
 
+const selectDefaultStakingDataByValidatorId = createSelector(
+  selectAssetIdParamFromFilterOptional,
+  selectSupportsCosmosSdkParamFromFilterOptional,
+  selectValidators,
+  (assetId, supportsCosmosSdk = true, stakingDataByValidator) => {
+    if (supportsCosmosSdk || !assetId) return null
+
+    const { chainId } = fromAssetId(assetId)
+
+    const defaultValidatorAddress = (() => {
+      switch (chainId) {
+        case cosmosChainId:
+          return SHAPESHIFT_COSMOS_VALIDATOR_ADDRESS
+        case osmosisChainId:
+          return SHAPESHIFT_OSMOSIS_VALIDATOR_ADDRESS
+        default:
+          return ''
+      }
+    })()
```

Comment:
We've used this chunk twice, we could abstract it as a pure utility function.

Reply:
Since this is already accepted, will stack it - that will also be more atomic since I spotted two other places in current develop with a similar implementation

Reply:
https://github.com/shapeshift/web/pull/2560